discord-bot/src/main/java/fr/pacifista/bot/discord/commands/CommandTicket.java

```diff
@@ -0,0 +1,94 @@
+package fr.pacifista.bot.discord.commands;
```

Comment:
Pourquoi la PR touche aux tickets ?

Reply:
L'ajout de sous commandes (subcommands) touche toute les commandes, y compris celles des tickets et la "/ip". Ca m'a permis de réutiliser une sous-feature des subcommands dans les tickets (en y repensant c'est pas ouf de mettre ca comme ca, my bad)
C'est mieux de laisser ou de remettre comme avant ?

Reply:
Nan nan is okai c'est de la refacto !